.chloggen/2528.yaml

```diff
@@ -0,0 +1,7 @@
+change_type: enhancement
+component: gen-ai
+note: | 
+  Add new attributes to 'invoke_agent' and 'execute_tool' span to capture input and output definition for Multi-Agentic systems.
```

Comment:
This only says Multi-Agentic, but it applies to single agents too, right?

Reply:
yes @alexmojaki . Please allow me to edit it to Single and Multi-Agent systems